src/components/PostPageDetail/index.js

```diff
@@ -730,7 +731,7 @@ const styles = StyleSheet.create({
     paddingHorizontal: 0,
     width: '100%',
     borderBottomWidth: 1,
-    borderBottomColor: '#C4C4C4'
+    borderBottomColor: COLORS.balance_gray
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Replacing hardcoded color values with centralized color constants improves maintainability and consistency of the codebase.
```diff
- borderBottomColor: '#C4C4C4'
+ borderBottomColor: COLORS.balance_gray
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->